Google-Summer-of-Code-2025.md

```diff
@@ -43,6 +43,18 @@ Furthermore, note that GSoC isn't just about code -- it's also about communicati
 You might also enjoy the "weekly journals" from some of our previous contributors: **[Rd4dev](https://medium.com/@rd4dev)** and **[@theMr17](https://medium.com/@Mr_17)**.
 
 
+## Selected Projects
+
+The following projects (with linked proposals) are selected as part of GSoC 2025:
+
+* [Mohit](pdfs/GSoC25MohitRuwatia.pdf): Clean up the structure for study guides and worked examples. Mentor: Kevin.
+* Hardik (TODO: PDF will be added soon): Lesson player redesign. Mentor: Amy.
+* [Jayam](pdfs/GSoC2025Jayam.pdf): Acceptance tests. Mentor: Christie Ho.
```

Comment:
If we are just going by first name then the mentor should be Christie.

If we are using full names then let's do that for everyone.